app/assets/stylesheets/puppetclasses.scss

```diff
@@ -13,6 +13,44 @@
   }
 }
 
+.matcher-field-changed {
```

Comment:
can we please use a different css file ? I'd rather see a new one specific for `lookup_keys`, this one is going to get removed with puppet, but this addition should stay in core.